saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/authentication/Saml2AuthenticationRequestContext.java

```diff
@@ -87,6 +113,41 @@ public String getRelayState() {
 		return this.relayState;
 	}
 
+	/**
+	 * Returns the desired ({@code AuthnContextClassRef} that this SP wishes to set in the
+	 * request.This property populates the
+	 * {@code AuthNRequest.RequestedAuthnContext.AuthnContextClassRef} XML attribute.
+	 * @return the AuthnContextClassRef value
+	 * @since 5.5
+	 */
+	public List<String> getRequestAuthnContextRefs() {
+		return this.requestAuthnContextRefs;
+	}
+
+	/**
+	 * Returns the isPassive value, if present in the parameters
+	 * @return the isPassive value, default is set to false
+	 * @since 5.5
+	 * @see <a href=
+	 * "https://wiki.shibboleth.net/confluence/display/SP3/isPassive">OpenSAML 3
+	 * isPassive</a>
+	 */
+	public boolean getIsPassive() {
```

Comment:
Thinking about this just a bit more, it seems like there's value in having these use `Boolean` instead of `boolean`.

The reason for that is so that an application can tell Spring Security that it wants *no value* for an attribute by setting it to `null`.

Reply:
Yup, that is a great point.